### ISSUE TITLE
Wait on moderator.. does not notice moderators online.

### ISSUE THREAD

**Issue by nomisge** [blindsidenetworks/moodle-mod_bigbluebuttonbn#515]
I run the latest release of this plugin (v2.5-alpha.1) on a up to date moodle 3.11.
When I start a meeting as teacher I become a moderator and it does show that there is a moderator.
If I have the setting "wait on moderator" set, no student can join.
If I unset that option they can join. If I elevate someone to moderator during the meeting, it shows in the plugin.

But no matter what case it also says: "No one is in this meeting :-( Moderators" (or in German: "Niemand ist in diesem Meeting. :-( Moderator/innen)

![grafik](https://user-images.githubusercontent.com/16142147/141147620-251ff85f-86f7-4bda-86b0-e2396cf9a1ca.png)

I guess somehow the code that outputs that line and takes care of managing who is allowed to join is broken.
If you tell me where to look in the code, I'm happy to help.

Thx for any hints in this ...

**Comment by nomisge** [comment]
Problem persists in Verison 3.0Alpha1.

**Comment by nomisge** [comment]
Done some more checks on a fresh non-production moodle server.

if I set the following in config.php
```
$CFG->bigbluebuttonbn_waitformoderator_default = 1;
$CFG->bigbluebuttonbn_waitformoderator_editable = 0;
```
anyone can join on a fresh bigbluebuttonbn instance, even if there is no moderator. The setting waitformoderator_default seems to be completely ignored.

But if I set the following in config.php
```
$CFG->bigbluebuttonbn_waitformoderator_default = 1;
$CFG->bigbluebuttonbn_waitformoderator_editable = 1;
```
the setting waitformoderator_default still seems to be ignored, but I can then set the setting from within the instance and I get an error "Waiting for a moderator to join." with a link to https://docs.moodle.org/311/de/error/mod_bigbluebuttonbn/waitformoderator

Same happens if set aforementioned settings not from config.php but from the plugins settings in the administration.
Since the issue changed a bit, I'm going to open a new issue.
